concent_api/gatekeeper/tests/test_gatekeeper_views.py

```diff
@@ -387,6 +398,28 @@ def test_upload_should_return_401_if_file_categories_are_not_unique_across_file_
         self.assertEqual("application/json", response["Content-Type"])
         self.assertEqual(response.json()["error_code"], ErrorCode.MESSAGE_FILES_CATEGORY_NOT_UNIQUE.value)
 
+    @freeze_time("2018-12-30 11:00:00")
+    def test_upload_should_return_401_if_invalid_protocol_version(self):
```

Comment:
is `GOLEM_MESSAGES_VERSION` relevant for file upload?
@rwrzesien ?
@bartoszbetka ?

Reply:
@dybi Yes, please check the blueprint's comments https://github.com/golemfactory/concent/issues/857 about `gatekeeper`.